frontend/src/Pages/ResetPassPage.js

```diff
@@ -0,0 +1,28 @@
+import { Container, Box, Text } from "@chakra-ui/react";
+import ResetPassword from "../components/Authentication/ResetPassword";
+
+function ResetPasswordPage() {
+  return (
+    <Container maxW="xl" centerContent>
```

Comment:
Why forget and reset password is different?

Reply:
forget-password is to take email as input and then after verifying the email exists in db, it sends the otp to that email.
reset-password is to verify otp and then take new password as input from user. And after sending the email, I am navigating to the rest-pass page from forget-pass page.

Reply:
ok